apim-apk-agent/internal/synchronizer/apis_fetcher.go

```diff
@@ -151,6 +153,47 @@ func FetchAPIsOnEvent(conf *config.Config, apiUUIDList []string, k8sClient clien
 				// Todo: Read the apis.zip and extract the api.zip,deployments.json
 			}
 
+			if err != nil {
+				logger.LoggerMsg.Error("Error while reading zip", err)
+			}
+
+			artifact, decodingError := transformer.DecodeAPIArtifact(data.Resp)
+
+			if decodingError != nil {
+				logger.LoggerSync.Error("Error while decoding the API Project Artifact:", decodingError)
+				return
+			}
+
+			apkConf, _, apkErr := transformer.GenerateAPKConf(artifact.APIJson)
+			logger.LoggerSync.Infof(apkConf)
```

Comment:
Is it added for debugging purposes? If so please remove. If not It should be a debug log and we should add description to the log

Reply:
This is also used for the debugging and now it's removed